tests/reconfig/results/1.1.0/1.1.0.md

```diff
@@ -0,0 +1,85 @@
+| 1           | 150          | 3.5                  | 1                        | 2             | 272.5                      | 100%     | 100%      |
+| 2           | 30           | 34                   | <1                       | 93            | 136                        | 100%     | 100%      |
+| 2           | 150          | 176.5                | <1                       | 451           | 203.98                     | 100%     | 100%      |
+| 3           | 30           | <1                   | 1                        | 93            | 125.7                      | 100%     | 100%      |
+| 3           | 150          | 1                    | 1                        | 453           | 126.71                     | 100%     | 100%      |
+
+
+### Event Batch Processing
+
+| Test number | NumResources | Event Batch Total | Event Batch Processing avg time (ms) | <= 500ms | <= 1000ms | <= 5000ms | <= 10000ms | <= 30000ms |
+|-------------|--------------|-------------------|--------------------------------------|----------|-----------|-----------|------------|------------|
+| 1           | 30           | 70                | 5.12                                 | 100%     | 100%      | 100%      | 100%       | 100%       |
+| 1           | 150          | 309               | 2.14                                 | 100%     | 100%      | 100%      | 100%       | 100%       |
+| 2           | 30           | 442               | 35.4                                 | 100%     | 100%      | 100%      | 100%       | 100%       |
+| 2           | 150          | 2009              | 54.76                                | 100%     | 100%      | 100%      | 100%       | 100%       |
+| 3           | 30           | 373               | 35.72                                | 99.73%   | 99.73%    | 100%      | 100%       | 100%       |
+| 3           | 150          | 1813              | 39.46                                | 99.94%   | 99.94%    | 99.94%    | 99.94%     | 100%       |
+
+> Note: The outlier for test #3 is the event batch that contains the Gateway. It took ~13s to process.
+
+## NumResources to Total Resources
+
+| NumResources | Gateways | Secrets | ReferenceGrants | Namespaces | application Pods | application Services | HTTPRoutes | Total Resources |
+|--------------|----------|---------|-----------------|------------|------------------|----------------------|------------|-----------------|
+| x            | 1        | 1       | 1               | x+1        | 2x               | 2x                   | 3x         | <total>         |
+| 30           | 1        | 1       | 1               | 31         | 60               | 60                   | 90         | 244             |
+| 150          | 1        | 1       | 1               | 151        | 300              | 300                  | 450        | 1204            |
```

Comment:
I found a typo in [cafe-routes.yaml](https://github.com/nginxinc/nginx-gateway-fabric/blob/19a1e27b37ed88f45c38b8524b3955a706327b04/tests/reconfig/scripts/cafe-routes.yaml#L7). The parentRef name is `gateway.networking.k8s.io/v1` instead of `gateway`. This means that the `cafe-tls-redirect` route is ignored by NGF, and only 300/450 routes are attached to the gateway.

I didn't find it until I ran the last test, and since it exists in the 1.0 test, I chose to leave it as is so we can compare the results. 

If we want to remove it, we will have to re-run the test suite against the 1.0 release and edge.

Reply:
If we feel it's worth having in the future, maybe we just start doing it for 1.2 going forward, knowing that the current results will be a bit different.